packages/neos-ui-ckeditor5-bindings/src/cleanupContentBeforeCommit.spec.js

```diff
@@ -8,3 +8,28 @@ test('remove empty nbsp', () => {
     assertCleanedUpContent('<p>&nbsp;</p>', '');
     assertCleanedUpContent('<span>&nbsp;</span>', '');
 })
+
+describe('ckeditor DisabledAutoparagraphMode hack, cleanup outer spans', () => {
+    test('noop', () => {
+        assertCleanedUpContent('<p></p>', '<p></p>');
+
+        assertCleanedUpContent('', '');
+
+        assertCleanedUpContent('<span><span>foo</span></span>', '<span><span>foo</span></span>');
+    })
+
+    test('cleanup single root <span>', () => {
+        assertCleanedUpContent('<span></span>', '');
+        assertCleanedUpContent('<span>foo</span>', 'foo');
+    })
+
+
+    test('cleanup multiple root <span>', () => {
+        assertCleanedUpContent('<span>foo</span><span>bar</span>', '<span>foo</span><span>bar</span>');
+    })
+
+    test('cleanup <span> root after other root', () => {
+        // in the case you had multiple paragraphs and a headline and switched to autoparagrahp: false
```

Comment:
```suggestion
        // in the case you had multiple paragraphs and a headline and switched to autoparagraph: false
```

Reply:
i was waiting for it to spell it wrong ^^